### PR TITLE
Update GoFileRoom connector files in repo 

### PR DESCRIPTION
**Pull request by AlfredoDinunzioTR**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by calvinleong** [comment]
Can this pass the swagger and change validation step? This updated response is valid for our API and we have tested. For any existing customers using this our support team can handle.

File: certified-connectors/gofileroom/apiProperties.json [hunk 1]
```diff
@@ -1,80 +1,93 @@
...
+            "RouteWorkflowV2",
+            "TrackingReportByWorkflow",
+            "AddMasterDeliverable",
+            "GetUserInfoV2",
+            "TrackingReportByDeliverableV2"
+          ]
+        }
+      }
+    ],
+    "publisher": "Teribery, Dustin (Tax&Accounting Prof)",
```
Comment (sriyen-msft) [inline certified-connectors/gofileroom/apiProperties.json:90]: @AlfredoDinunzioTR The given publisher name looks like an individual name which is not allowed. It should be an organization or company name for "certified-connectors"

File: certified-connectors/gofileroom/settings.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (sriyen-msft) [inline certified-connectors/gofileroom/settings.json:1]: @AlfredoDinunzioTR The settings.json file is not required, Please remove it from the PR.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by AlfredoDinunzioTR — COMMENTED** [review]
requested updates have been completed

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**